py/trtorch/ptq.py

```diff
@@ -6,7 +6,11 @@
 from trtorch._compile_spec import _parse_compile_spec
 from trtorch._version import __version__
 from types import FunctionType
-import tensorrt as trt
+
+ENTROPY_CALIBRATION = trtorch._C.CalibrationAlgo.ENTROPY_CALIBRATION

```

Comment:
Is it possible to group these in an enum or something called `CalibrationAlgo`? I think we do something similar for logging

Reply:
Done. Is the new class okay or any better way to expose them ?